src/Exceptionless.Api/Controllers/AuthController.cs

```diff
@@ -146,6 +146,27 @@ public class AuthController : ExceptionlessApiController {
             return Ok(new TokenResult { Token = await GetTokenAsync(user) });
         }
 
+        [ApiExplorerSettings(IgnoreApi = true)]
+        [HttpGet]
+        [Route("logout")]
+        [Authorize(Roles = AuthorizationRoles.User)]
+        public async Task<IHttpActionResult> LogoutAsync() {
+            if (!User.IsTokenAuthType())
+                return Ok();
+
+            var id = User.GetLoggedInUsersTokenId();
```

Comment:
I don't really like this name, but it's the token id if you logged in via a user auth work flow